doc/sphinx/source/community/release_strategy.rst

```diff
@@ -508,7 +498,7 @@ by the actual ID.
 
 Changelog
 ---------
-
+- 2022-07-21 Update instructions to follow the new release procedure
```

Comment:
Not sure if this is necessary - the previous comment says "future changes tracked by git".

Reply:
Sure, I must have missed that comment